### PR TITLE
Refactoring and new features - (json-web-tokens)

### PR DESCRIPTION
**Pull request by ozomer**
Enabling bcrypt password hashes, replacing cookies with json-web-tokens (based on a cryptographically random key) that are saved in sessionStorage, handling errors.

I did not test all edge cases (loading/saving to configuration, default-connections, etc.), but I tried to keep existing working code.

Backward compatibility is maintained to nodejs 6.* only. This version uses arrow functions and template strings.

### CODE REVIEW

**Comment by ozomer** [comment]
Cool, that was quick.
When you publish to npm, I suggest to bump the version to 0.5, since it is no longer fully backward compatible for nodejs versions smaller than 6.0 (template-strings & arrow functions).
I still avoided async-await...

Furthermore, notice that I changed the package.json dependencies to have exact values (using `npm i --save --save-exact`). It's a good habit to prevent production servers from crashing just because someone on the other side of the world pushed a bug...

Oh, and it's about time to switch from express 3 to express 4, but this is going to be a headache...

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "Alex Ehrnschwender <alexanderscott46@gmail.com>",
     "Matthew Scragg <scragg@gmail.com>"
   ],
-  "name": "redis-commander",
+  "name": "@awearsolutions/redis-commander",
```
Comment (VFedyk) [inline package.json]: @ozomer is this correct package name? I'm asking because it was merged to `master`.

Comment (ozomer) [reply package.json]: Yes, I have cloned the project to [here](https://bitbucket.org/awearsolutionsltd/redis-commander/overview) because I needed some fixes for some internal project.
Feel free to merge the code (I don't know about the required git commands, but I guess copy-pasting the code will do the trick).
